projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/read-linked-cases-field.component.spec.ts

```diff
@@ -0,0 +1,551 @@
+                })
+              ],
+              show_condition: 'PersonFirstName="Janet"'
+            },
+            {
+              id: 'HistoryTab',
+              label: 'History',
+              order: 1,
+              fields: [Object.assign(new CaseField(), {
+                id: 'CaseHistory',
+                label: 'Case History',
+                display_context: 'OPTIONAL',
+                field_type: {
+                  id: 'CaseHistoryViewer',
+                  type: 'CaseHistoryViewer'
+                },
+                order: 1,
+                value: null,
+                show_condition: '',
+                hint_text: ''
+              })],
+              show_condition: ''
+            },
+            {
+              id: 'SomeTab',
+              label: 'Some Tab',
+              order: 3,
+              fields: [],
+              show_condition: ''
+            }
+          ]
+        }
+      }
+    }
+  };
+
+  const mockRouter = {
+    navigate: jasmine.createSpy('navigate'),
+    url: ''
+  };
+
+  const linkedCases: CaseLink[] = [
+    {
+      caseReference: '1682374819203471',
+      reasons: [],
+      createdDateTime: '',
+      caseType: 'SSCS',
+      caseTypeDescription: 'SSCS case type',
+      caseState: 'state',
+      caseStateDescription: 'state description',
+      caseService: 'Tribunal',
+      caseName: 'SSCS 2.1'
+    },
+    {
+      caseReference: '1682897456391875',
+      reasons: [],
+      createdDateTime: '',
+      caseType: 'SSCS',
+      caseTypeDescription: 'SSCS case type',
+      caseState: 'state',
+      caseStateDescription: 'state description',
+      caseService: 'Tribunal',
+      caseName: 'SSCS 2.1'
+    }
+  ];
+
+  const linkCaseReasons: LovRefDataByServiceModel = {
+    list_of_values: [
+      {
+        key: 'progressed',
+        value_en: 'Progressed as part of this lead case',
+        value_cy: '',
+        hint_text_en: 'Progressed as part of this lead case',
+        hint_text_cy: '',
+        lov_order: 1,
+        parent_key: null,
+        category_key: 'caseLinkReason',
+        parent_category: '',
+        active_flag: 'Y',
+        child_nodes: null,
+        from: 'exui-default',
+      },
+      {
+        key: 'bail',
+        value_en: 'Bail',
+        value_cy: '',
+        hint_text_en: 'Bail',
+        hint_text_cy: '',
+        lov_order: 2,
+        parent_key: null,
+        category_key: 'caseLinkReason',
+        parent_category: '',
+        active_flag: 'Y',
+        child_nodes: null,
+        from: 'exui-default',
+      },
+      {
+        key: 'other',
+        value_en: 'Other',
+        value_cy: '',
+        hint_text_en: 'Other',
+        hint_text_cy: '',
+        lov_order: 3,
+        parent_key: null,
+        category_key: 'caseLinkReason',
+        parent_category: '',
+        active_flag: 'Y',
+        child_nodes: null,
+        from: 'exui-default',
+      },
+    ]
+  };
+
+  linkedCasesService = {
```

Comment:
@johnbenjamin-hmcts - could be declared immediately on Line 20

Reply:
@Daniel-Lam I have updated the code now.